docs/source/pillars/computing_technology.md

```diff
@@ -269,6 +272,9 @@ _The ability of the TRE organisation to identify, maintain, and verify informati
 * - You must be able to replace a non-compliant TRE with a compliant system.
   - This might involve reconfiguring a running system or by replacing it with a compliant one.
   - Mandatory
+* - You must ensure the TRE has a process in place for applying security updates for software used for configuring and running infrastructure, as well as any anti-virus/malware.
+  - This includes any software used for remote desktop portals, databases, webapps, creating and destroying compute infrastructure, configuration management, or software used for monitoring the TRE.
```

Comment:
Echoing @jemrobinson's point about turtles all the way down here.

The way it is written says to me "If you deploy/configure the TRE from your personal computer, the TRE organisation is responsible for enforcing software updates and anti-virus on that system".

For example, DSH at the moment would fail this requirement.

Reply:
@JimMadge do you think James' updated wording works?